### PR TITLE
Log less in AMP output

### PR DESCRIPTION
**Pull request by kristoferbaxter**
Resolves https://github.com/ampproject/amphtml/issues/28714

### CODE REVIEW

File: src/worker-thread/index.amp.ts [hunk 1]
```diff
@@ -61,6 +61,8 @@ import { initialize } from './initialize';
 import { wrap as longTaskWrap } from './long-task';
 import { callFunctionMessageHandler, exportFunction } from './function';
 
+declare const WORKER_DOM_DEBUG: boolean;
```
Comment (samouri) [inline src/worker-thread/index.amp.ts:64]: is there any location we could put this such that we'd only need to declare it once?

Comment (kristoferbaxter) [reply src/worker-thread/index.amp.ts:64]: Good point, I'll address.

Comment (kristoferbaxter) [reply src/worker-thread/index.amp.ts:64]: Moved to `worker-thread.d.ts` like defined for `main-thread.d.ts`.

Perhaps in the future we should move this to a common definition file for both threads.

Comment (samouri) [inline src/worker-thread/index.amp.ts:64]: I think you forgot to delete this line

**Pull request merged**